UI overlook changes

HarshiSharma opened the pull request:
UI Changes for:
- Vertical Nav Bar
- Colour Scheme changes to make it more accessible (https://davidmathlogic.com/colorblind/#%23D81B60-%231E88E5-%23FFC107-%23004D40)
- Manage Page UI changes to add table and action button
<img width="1440" alt="Screenshot 2024-03-24 at 5 23 21 PM" src="https://github.com/VathsalaAchar/MLA-pilot-group4/assets/45699555/ac6bf2e4-8fc2-4955-b69b-727279c7c02a">
<img width="1440" alt="Screenshot 2024-03-24 at 5 23 37 PM" src="https://github.com/VathsalaAchar/MLA-pilot-group4/assets/45699555/7a3b5a5b-2d56-4d0d-b9be-5a81f7956a77">
<img width="1440" alt="Screenshot 2024-03-24 at 5 05 44 PM" src="https://github.com/VathsalaAchar/MLA-pilot-group4/assets/45699555/fa379faa-8111-47c7-84b4-99f1afc63e45">
<img width="1440" alt="Screenshot 2024-03-24 at 5 21 03 PM" src="https://github.com/VathsalaAchar/MLA-pilot-group4/assets/45699555/08b942d0-c199-4739-a287-2e5d94ffd19e">

HarshiSharma committed:
nav change manage ui change colour change and journal icon change

HarshiSharma committed:
removing dummy data part oops

LakshmiAvanthiKaneti reviewed (approved):
Nice work @HarshiSharma especially navbar and colour choice, even manage page looks good.

ruchika512 committed:
added track distance changes (#21)

Modified below services

Frontend service
Added the new field on UI and passed the field to activity-tracking web service

Validation to enter distance for running and swimming activities

Css changes to add style for distance input field

 modified:   frontend/src/App.css
 modified:   frontend/src/components/trackExercise.js
Activity-Tracking service
Added the new field distance in the schema

Added the new field in POST request & PUT request

  modified:   activity-tracking/models/exercise.model.js
  modified:   activity-tracking/routes/exercises.js

VathsalaAchar reviewed (approved):
This is brilliant! Great work @HarshiSharma

VathsalaAchar commented:
Holding off on merge till the login page is checked for the logo.

HarshiSharma committed:
login and signup UI fix

HarshiSharma committed:
login and signup UI fix

HarshiSharma committed:
merge

HarshiSharma committed:
Merge branch 'main' into ui-overlook-changes

HarshiSharma committed:
bracket fix
